benches/bloom_bench.rs

```diff
@@ -0,0 +1,64 @@
+use bytes::{BufMut, BytesMut};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use slatedb::filter::{filter_hash, BloomFilterBuilder};
+fn bloom_filter_insert_benchmark(c: &mut Criterion) {
+    let mut builder = BloomFilterBuilder::new(10); // Set bits_per_key
+    let key_sz = std::mem::size_of::<u32>();
```

Comment:
we probably want to use a more realistic key size here (4 bytes seems rather small)

Reply:
> we probably want to use a more realistic key size here (4 bytes seems rather small)

Would this be better?
`let key_sz = std::mem::size_of::<u64>();`

Reply:
I think we should make it flexible (e.g. use a `Vec<u8>` to store the key) and default it to something larger. IMO 32 bytes would be reasonable.